src/IJ_mv/IJMatrix_parcsr.c

```diff
@@ -1501,7 +1501,7 @@ hypre_IJMatrixAddToValuesParCSR( hypre_IJMatrix       *matrix,
                   offd_data = hypre_CSRMatrixData(offd);
                   if (!big_offd_j) 
                   {
-                     big_offd_j = hypre_CTAlloc(HYPRE_BigInt, offd_i[hypre_CSRMatrixNumRows(offd)], HYPRE_MEMORY_HOST);
```

Comment:
@ulrikeyang Do you think we really need SHARED here? I hesitate to put more SHARED in...